docs/overview/office-add-in-availability.md

```diff
@@ -332,27 +332,39 @@ To work as expected, your Office Add-in might depend on a specific Office applic
 
 <table style="width:80%">
   <tr>
-    <th style="width:10%">Platform</th>
-    <th style="width:10%">Extension points</th>
-    <th style="width:20%">API requirement sets</th>
-    <th style="width:40%"><a href="/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets"><b>Common APIs</b></a></th>
+    <th>Platform</th>
+    <th>Extension points</th>
+    <th>API requirement sets</th>
+    <th><a href="/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets"><b>Common APIs</b></a></th>
   </tr>
   <tr>
     <td>Office on the web</td>
     <td>- Custom Functions</td>
-    <td>- <a href="/office/dev/add-ins/excel/custom-functions-requirement-sets">CustomFunctionsRuntime 1.1</a></td>
+    <td>
+      - <a href="/office/dev/add-ins/excel/custom-functions-requirement-sets">CustomFunctionsRuntime 1.1</a><br>
+      - <a href="/office/dev/add-ins/excel/custom-functions-requirement-sets">CustomFunctionsRuntime 1.2</a><br>
+      - <a href="/office/dev/add-ins/excel/custom-functions-requirement-sets">CustomFunctionsRuntime 1.3</a>
+    </td>
     <td></td>
   </tr>
   <tr>
     <td>Office on Windows<br>(connected to a Microsoft 365 subscription)</td>
     <td>- Custom Functions</td>
```

Comment:
Consider updating based on the name of the extension point: https://docs.microsoft.com/office/dev/add-ins/reference/manifest/extensionpoint#extension-points-for-excel-only
Applies in Excel and Custom Functions sections
```suggestion
    <td>- CustomFunctions</td>
```